src/napari_brainbow_diagnose/_read_csv_widget.py

```diff
@@ -0,0 +1,33 @@
+import napari
+
+import pandas as pd
+import numpy as np
+
+from magicgui import magicgui
+from magicgui import magic_factory
+import pathlib
+
+def rgb_to_xy(r, g, b):
```

Comment:
Maybe this function name could go to `_utils_channel_space.py`.
I also suggest to rename to something more specific that `rgb_to_xy` with something more explicit `rgb_to_maxwell_triangle`. You can also add a small docstring.